packages/gatsby/src/query/query-watcher.js

```diff
@@ -220,11 +221,22 @@ const debounceCompile = _.debounce(() => {
   updateStateAndRunQueries()
 }, 100)
 
-const watch = rootDir => {
+const watch = async rootDir => {
   if (watcher) return
 
+  const modulesThatUseGatsby = await getGatsbyDependents()
+
+  const packagePaths = modulesThatUseGatsby.map(module => {
+    const filesRegex = `*.+(t|j)s?(x)`
+    const pathRegex = `/{${filesRegex},!(node_modules)/**/${filesRegex}}`
+    slash(path.join(module.path, pathRegex))
```

Comment:
Shouldn't this be:
```suggestion
    return slash(path.join(module.path, pathRegex))
```
?

Reply:
Oops